app/public/css/styles.css

```diff
@@ -2521,58 +2524,93 @@ li.main-tab input + label.selected:after {
 /* ADDRESSES */
 
 .addresses {
+  width: 100%;
   position: relative;
-  padding-right: 28px;
-}
-
-.addresses::before {
-  position: absolute;
-  content: '';
-  left: 56px;
-  right: 40px;
-  bottom: 40px;
-  height: 56px;
-  background-image: linear-gradient(
-    to top,
-    rgb(255, 255, 255),
-    rgba(255, 255, 255, 0.4)
-  );
-  z-index: 2;
 }
 
 .addresses-title {
   margin-bottom: 24px;
 }
 
 .addresses-content {
-  margin-bottom: 19px;
-  max-height: 736px;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  -webkit-overflow-scrolling: touch;
   position: relative;
 }
 
 .address {
-  padding: 24px 16px 24px 0;
+  padding: 24px;
   border-top: 1px solid var(--color-border);
 }
 
 .address:last-child {
-  margin-bottom: 8px;
+  padding-bottom: 0px;
+}
+
+.address .label {
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
 }
 
-.address-value {
+.address .value {
+  display: flex;
   color: var(--color-grey);
   word-break: break-all;
-  margin-bottom: 16px;
+}
+
+.address.expanded {
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+  background-color: var(--color-selected);
+  transition: background-color 0.25s ease-out 0.5s, box-shadow 0.25s ease-out 0.5s;
+}
+
+.address:not(.expanded) {
+  cursor: pointer;
+  transition: background-color 0.25s ease-out, box-shadow 0.25s ease-out;
+}
+
+.address .expanded {
+  display: flex;
+  margin-left: 32px;
+}
+
+.address .explorer-links {
+  margin-left: 38px;
+}
+
+.address .explorer-links.show {
+  max-height: 24px;
+  overflow: hidden;
+  transition: max-height 0.5s ease-out;
+}
+
+.address .expanded.show {
+  max-height: 208px;
+  overflow: hidden;
+  transition: max-height 0.5s ease-out;
+}
+
+.address .hide {
+  max-height: 0px;
+  overflow: hidden;
+  transition: max-height 0.5s ease-out;
+}
```

Comment:
I played around with the transitions a little bit, 0.5s seems a little laggy and slow. I tried 0.25s, also `transition: background-color 0.25s ease-out 0.25s, box-shadow 0.25s ease-out 0.25s;`  and it felt better for me, what do you think?

Reply:
Sounds good, changed it to 0.25s

Reply:
`transition: max-height 0.5s ease-out;` in 0-767px.css needs to be changed as well, now it behaves strangely on mobile

Reply:
Fixed